website_sale_stock_provisioning_time/views/website_sale_product.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template
+    id="product_website_sale_privious"
+    name="product_sale"
+    inherit_id="website_sale.product"
+  >
+        <xpath expr="//a[@id='add_to_cart']" position="after">
+            <div class="add_atribute" style="margin-top:10px;">
+                <t t-if="product.provisioning_time_in_workdays">
+                    <span>Provisioning time until shipment:</span>
+                    <span t-esc="product.provisioning_time_in_workdays" />
+                    Work days
```

Comment:
if the field has value 1 for the product, the output will be "1 Work days" and that is grammatically not right.

Reply:
I'll fix it